### DISCUSSION TITLE
shortcut to copy a record

### DISCUSSION THREAD

**Discussion by josejachuf** [emmett-framework/emmett#393] [Q&A] [answered]
There is a shortcut to copy / clone a record in the ORM?

Something like:

> row = Mymodel.get (id=1)
> Mymodel.create(row)

José

**Comment by gi0baro** [comment, marked as answer]
Not really, at the moment it will require some "manual" steps. But we can add some facilities in 2.4

**Comment by josejachuf** [comment]
Hi @gi0baro 
This was implemented? And if so, how is it used?
Thanks!

**Reply by gi0baro** [reply]
@josejachuf sorry for the late reply! No unfortunately it is not. But maybe you can open up an issue for a feature request and maybe we can implement it in 2.6
